SpiceSharp/Components/Currentsources/CurrentControlledCurrentsource/CurrentControlledCurrentsource.cs

```diff
@@ -54,7 +54,7 @@ static CurrentControlledCurrentsource()
         /// Constructor
         /// </summary>
         /// <param name="name">The name of the current controlled current source</param>
-        public CurrentControlledCurrentsource(CircuitIdentifier name) : base(name)
+        public CurrentControlledCurrentsource(CircuitIdentifier name) : base(name, 2)
```

Comment:
@svenboulanger  Move `2` to const of the CurrentControlledCurrentsource. It would be good to know from outside what is the number of connections/pins

Reply:
Same for other components

Reply:
The number of pins can be found using `PinCount` defined in CircuitComponent.cs.

Reply:
ok